### ISSUE TITLE
error: ./config/window.lua:175: attempt to index field 'view' (a nil value)

### ISSUE THREAD

**Issue by Ygrex** [aidanholm/luakit#147]
found no stable way to reproduce but it sometime falls in an infinite loop:

```
[190237.906369] W: luaH_dofunction:157: error while running function
stack traceback:
error: ./config/window.lua:175: attempt to index field 'view' (a nil value)
```

1d24f0d build with default config

luakit continues working but stderr'ing the same message again and again until restarted

**Comment by aidanholm** [comment]
That's a new one. It's attempting to emit a signal on the current webview widget, which is set to `nil` during a tab switch. But attempting to access the current webview if it's `nil` should automatically update it to the right value.

The error message spam is because idle functions that error out aren't cancelled. #148

**Comment by aidanholm** [comment]
@Ygrex Are you still experiencing this issue?

**Comment by Ygrex** [comment]
yes as of 8916bfa (March 12th):
```
[ 4341.601666] W: luaH_dofunction:157: error in ./lib/lousy/widget/tablist.lua:37: attempt to index field '?' (a nil value)
stack traceback:
[ 4341.601695] W: luaH_dofunction:157: error in ./config/window.lua:157: attempt to index field 'view' (a nil value)
stack traceback:
```
will try the updated one and let you know

**Comment by aidanholm** [comment]
OK, I can reproduce it! Happens when you close windows with more than one tab. Should have it fixed soon.

**Comment by aidanholm** [comment]
Fixed at 373c0bfe! Turned out to be mostly harmless, but good to clean it up anyway.
